tests/test_base.py

```diff
@@ -689,6 +689,17 @@ async def foo():
             asyncio.wait_for(foo(), timeout=float('inf')))
         self.assertEqual(res, 123)
 
+    def test_shutdown_default_executor(self):
+        if not hasattr(self.loop, "shutdown_default_executor"):
+            raise unittest.SkipTest()
+
+        async def foo():
+            await self.loop.run_in_executor(None, time.sleep, .1)
+
+        self.loop.run_until_complete(foo())
+        self.loop.run_until_complete(
+            self.loop.shutdown_default_executor())
```

Comment:
Is this the only test we have in cpython/asyncio/tests?

Reply:
Actually, it looks like [this](https://github.com/python/cpython/blob/9650fe0197779b4dfded94be111e39c5810f098f/Lib/test/test_asyncio/test_events.py#L358-L376) is the only test that uses it in CPython:

```py
    def test_run_in_executor_cancel(self):
        called = False

        def patched_call_soon(*args):
            nonlocal called
            called = True

        def run():
            time.sleep(0.05)

        f2 = self.loop.run_in_executor(None, run)
        f2.cancel()
        self.loop.run_until_complete(
                self.loop.shutdown_default_executor())
        self.loop.close()
        self.loop.call_soon = patched_call_soon
        self.loop.call_soon_threadsafe = patched_call_soon
        time.sleep(0.4)
        self.assertFalse(called)
```

In retrospect, I probably should have added some dedicated test(s) for `shutdown_default_executor()` in the PR that added it in the first place, but at the time, I didn't have a decent understanding of asyncio unit tests. Should I add this one to the CPython regression tests and maybe a couple more if I can think of some that could be useful?

Reply:
> Should I add this one to the CPython regression tests and maybe a couple more if I can think of some that could be useful?

The more tests the better :) If they are not redundant, of course. I'll merge this PR now, huge thanks!